test/raap.txt

```diff
@@ -0,0 +1,5 @@
+# Specify the class/module and method names to be executed by RaaP.
+# By prefixing with `!`, you can skip testing a method.
+# Lines beginning with `#` are comments.
+
+Set[Integer]
```

Comment:
We can add more testing.

Reply:
Ideally, RaaP should be able to automatically test with the given bound (`Object` at this writing), and maybe also a variety of other sample classes.
Then, this file would simply be
```suggestion
Set
```

Reply:
The minimal condition to satisfy the upper bound is `Object`, but using Object in Set makes it harder to visually interpret the reproducible code, so I use `Integer` instead.